src/app/components/accounts/profile-details/profile-details.component.ts

```diff
@@ -41,24 +42,24 @@ export class ProfileDetailsComponent implements OnInit {
         });
     }
 
-    onSubmit(FormData) {
+    onSubmit(FormData: unknown): void {
```

Comment:
same here

Reply:
Do you know the type? I tried doing a map and it said that unknown should be used instead.